_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -136,11 +136,13 @@ function page_render() {
 		/** This action is already documented in views/admin/admin-page.php */
 		do_action( 'jetpack_notices' );
 
-		echo file_get_contents( JETPACK__PLUGIN_DIR . '_inc/build/static.html' );
+		$static_html = wp_remote_get( JETPACK__PLUGIN_URL . '/_inc/build/static.html', array( 'sslverify' => false ) );
+		echo 200 == wp_remote_retrieve_response_code( $static_html ) ? wp_remote_retrieve_body( $static_html ) : esc_html__( 'Error fetching page.', 'jetpack' );
```

Comment:
Minor nitpick: can we please avoid long lines and carry stuff over after `?` here and in the diffs below this one? Thank you!